s5_base/run.sh

```diff
@@ -6,148 +6,271 @@
 #SBATCH --output=logs/samrun%J.out
 #SBATCH --nodelist=terra
 
+# To be run from the s5_base directory
+
 . ./cmd.sh
 . ./path.sh
 # setup the steps and utils directories
 . ./setup.sh
+. ./local/utils.sh
 
-set -euo pipefail
+set -eo pipefail
 
-samromur_root=samromur
+# NOTE! In the future the ASR data, LM training text and pronunciation dictionary
+# will be downloaded from online first, e.g. Clarin
+samromur_root=/data/asr/samromur/samromur_ldc #samromur
+lm_train=/models/samromur/rmh_2020-11-23_uniq.txt
+prondict_orig=/models/samromur/prondict_rmh_2020-12-02.txt
+prondict=data/prondict_w_samromur.txt
+g2p_model=../preprocessing/g2p/ipd_clean_slt2018.mdl
 stage=0
 
 . utils/parse_options.sh
 
 #TODO: set the samromur data and metadata variables within path.sh
-mfccdir=`pwd`/mfccs
-num_jobs=10
+mfccdir=mfcc
+num_jobs=20
+nj_decode=32
+localdict=data/local/dict
 
 if [ $stage -le 0 ]; then
-  echo "Create training data from samromur training"
-  local/samromur_data_prep.sh $samromur_root training data
-  local/samromur_data_prep.sh $samromur_root test data
-  local/samromur_data_prep.sh $samromur_root eval data
-
-  utils/combine_data.sh data/train data/samromur_training
-  utils/combine_data.sh data/dev data/samromur_test
-  utils/combine_data.sh data/eval data/samromur_eval
-  utils/fix_data_dir.sh data/train
-  utils/fix_data_dir.sh data/dev
-  utils/fix_data_dir.sh data/eval
+    echo "Create training data"
+    # NOTE! I would rather like "train" than "training"
```

Comment:
We're just using the convention found within the metadata_v2/metadata files, right? Although I also agree that I prefer train.